tests/jeos/firstrun.pm

```diff
@@ -114,6 +114,38 @@ sub verify_partition_label {
     script_output('sfdisk -l') =~ m/Disklabel type:\s+$label/ or die "Wrong partion label found, expected '$label'";
 }
 
+sub create_user_in_terminal {
+    return if script_run("getent passwd $username") == 0;
```

Comment:
IMO if the firstboot wizard was used to create the user, it should verify that the user was created instead of silently creating it.

Reply:
The verification itself happens later in the code, when `user-console` is selected. I have also added a flag and option to skip the user creation in UI by setting test suite variable.